src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1578,6 +1586,20 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
                 ->addTag('messenger.receiver', array('alias' => $name))
                 ->addTag('messenger.sender', array('alias' => $name))
             ;
+
+            if ('symfony://kernel.terminate' === $transport['dsn']) {
+                $busId = $transport['options']['bus'] ?? null;
+                if (null === $busId) {
+                    throw new InvalidArgumentException(sprintf('Missing mandatory "bus" option for transport "%s"', $name));
```

Comment:
Can you default on the bus name if only one is defined? It will very likely be most of the use-cases.